.github/workflows/deploy-prod.yml

```
@@ -54,15 +54,25 @@ jobs:
+        if: failure()
+        continue-on-error: true
+        uses: actions/upload-artifact@v4
         with:
-          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
-          packageManager: bun
-          # Cloudflare API truncates commit message at ~384 bytes without UTF-8 boundary check (workers-sdk#11749).
-          # Use an explicit ASCII-only message to avoid "Invalid commit message" (code 8000111).
-          command: pages deploy dist --project-name=zedi --commit-message "Deploy from GitHub Actions (main) ${{ github.sha }}"
+          name: wrangler-logs-frontend
+          path: ${{ runner.home }}/.config/.wrangler/logs/
```

actionlint (1.7.11)

[error] property "home" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)